preview/galleon-local/src/main/resources/modules/system/layers/base/org/hibernate/models/hibernate-models/main/module.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright The WildFly Authors
+  ~ SPDX-License-Identifier: Apache-2.0
+  -->
+
+<module xmlns="urn:jboss:module:1.9" name="org.hibernate.models.hibernate-models">
+
+    <properties>
+        <property name="jboss.api" value="private"/>
+    </properties>
+
+    <resources>
+        <artifact name="${org.hibernate.models:hibernate-models}"/>
+        <artifact name="${org.hibernate.models:hibernate-models-jandex}"/>
```

Comment:
@yrodiere @marko-bekhta I wasn't 100% sure if we should be using the `hibernate-models-jandex` or not. WildFly does use Jandex, so it seemed logical. Please let me know if this is wrong.

Reply:
I think this will result in this being executed:

https://github.com/hibernate/hibernate-models/blob/f66a5f10e7d42ebafe9853f384ba6201b1a0812e/hibernate-models-jandex/src/main/java/org/hibernate/models/jandex/spi/JandexBuildingContextProvider.java#L32-L36

Which, worst case, does nothing (if WildFly doesn't pass the Jandex index to Hibernate ORM), and best case uses a new Jandex-based model retrieval (better performance) or just breaks everything (we'll get feedback!).

So, +1 on doing this in WildFly Preview.